src/config/default-release-config.js

```diff
@@ -0,0 +1,23 @@
+export const defaultReleaseConfig = {
+  build: true,
+  types: true,
+  test: true,
+  lint: true,
+  contributors: true,
+  bump: true,
+  changelog: true,
+  publish: true,
+  commit: true,
+  tag: true,
+  push: true,
+  ghrelease: true,
+  docs: true,
+  ghtoken: '',
+  type: 'patch',
+  preid: undefined,
+  distTag: 'latest',
+  remote: 'origin',
+  siblingDepUpdateMessage: 'deps: update sibling dependencies',
+  siblingDepUpdateName: 'aegir[bot]',
+  siblingDepUpdateEmail: 'aegir[bot]@users.noreply.github.com'
```

Comment:
This config was odd to me.

As far as I can tell, most of them aren't used. But, the release command does a lot of exec-ing and I didn't trace the code all the way through. So left it as is just in case.